### PR TITLE
Add `RomoDate.daysDiff` helper

### PR DESCRIPTION
**Pull request by jcredding**
This adds a helper for calculating the number of days between
two dates. The helper converts the dates to their millisecond
time values, subtracts them, and then converts the difference to
a number of days by dividing it by the number of milliseconds in
a day. The helper rounds the difference to get an integer of the
days.

@kellyredding - Ready for review.

### CODE REVIEW

File: assets/js/romo/date.js [hunk 1]
```diff
@@ -78,6 +78,15 @@ var RomoDate = {
     return RomoDate.daysInMonth(d) - d.getDate() + 1;
   },
 
+  daysDiff: function(firstDate, secondDate) {
+    var diff = (firstDate.getTime() - secondDate.getTime()) / 864e5; // 1000 * 60 * 60 * 24
```
Comment (kellyredding) [inline assets/js/romo/date.js]: @jcredding did not know you could represent integers that way - cool!

Comment (kellyredding) [reply assets/js/romo/date.js]: @jcredding I'd prefer casting the given "dates" to actual dates like the method above does.  This keeps the api consistent and I think allows you to pass strings directly.

**Review by kellyredding — APPROVED** [review]
@jcredding a comment about casting but I'm good once that is done 👎

**Comment by jcredding** [comment]
@kellyredding - FYI I switched to rounding the difference to get an integer for the days instead of flooring it. Javascript floating point precision seems to get off randomly so you get a value of like `25.98333333` days when it should be `26`. This seems to fix all the issues in our app but I'll keep an eye on it going forward.

**Comment by kellyredding** [comment]
@jcredding ok, that makes sense.

**Pull request merged**